docs/ref/dotq.md

```diff
@@ -1892,6 +1896,10 @@ symw| 25436
 
 ## `Xf` (create file)
 
+!!! warning "Depreciated"
+
+    Depreciated since 4.1t 2022.03.25. Using resulting files could return file format errors since 3.6.
+
```

Comment:
```suggestion
!!! warning "Deprecated"

    Deprecated since 4.1t 2022.03.25. Using resulting files could return file format errors since 3.6.

```